repo-audit.yml

```diff
@@ -0,0 +1,26 @@
+# Checks configuration file. These are just some examples.
+# The order in which they are declared will be maintained.
```

Comment:
It is not important at all. I just thought of adding a comment about it, in case we were to think in the future about clever optimisations like cache for already HEAD'ed or fetched files, etc. At the moment the order doesn't matter at all.